posts/2020-06-08-announcing-the-moving-from-scala-2-to-scala-3-course.adoc

```diff
@@ -0,0 +1,40 @@
+
+When I launched the idea about this new course at Lunatech, I got two enthusiastic colleagues, _Neil Benn_ and _Robin Raju_, to join me in the effort. So, after about 2 months in elapsed time, we delivered the first edition of the two-day _"Moving from Scala 2 to Scala 3"_ course to a bunch of colleagues.
+
+Between the start of the project and the first delivery, the alpha version of the course was kindly and thoroughly reviewed by members of the Scala Center. We received a lot of very useful feedback which resulted in a series of corresponding updates to the course materials. Speaking of which: We have a series of 11 exercises, a course slide deck with approximately 140 slides, and a collection of code-snippets that can be used to demonstrate various features of the Dotty language treated in the course.
+
+Very important to note is that all these materials are open-sourced! You can find them in https://github.com/lunatech-labs/lunatech-scala-2-to-scala3-course[this public repository] on Github. This means that you can benefit from this at no extra cost but the effort you put into studying the course materials! _(Of course, should you wish to have this course taught to you in-class, ping us —although that will not be free of charge)_.
+
+== What's next...
+
+So, _"What's next?"_, you may ask. Well, in fact, quite a bit... Here's list of things that are in the works:
+
+* We plan to keep the course current with the release candidates of Dotty. At the time of writing, Dotty is at release 0.24.0-RC1 with 0.25 around the corner.
+* Robin is working on implementing CI/CD on the project. When that is done, we will automatically release so-called _"studentified"_  and _"linearized"_ versions of the course.
+
+In a nutshell, a _"studentified"_ version of the course is an artefact generated from the https://github.com/lunatech-labs/lunatech-scala-2-to-scala3-course/tree/exercises-master/exercises[course master repository]. This artefact is a self-contained sbt project that has special functionality built-in to provide a friendly learning experience. A student can navigate between different exercises and thereby pull in relevant test code automatically, read exercise instructions via a manual page command, save and restore the state of the code, and _"pull"_ the solution if required.
```

Comment:
Today I learned *artifact* can also be spelled *artefact*, and it's still technically correct. 🤔

Reply:
I guess this is a US- versus British English thing? My spelling checker always wants to correct it to _artefact_. I also didn't know the other spelling was valid until it was suggested to me...